src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -87,6 +87,16 @@
     <Compile Remove="runtime\PacketTracer\**\*.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' &lt; '5.0')">
```

Comment:
@adamsitnik I added the exclude condition to avoid using the directive which would need extra changes in future netcoreapp releases.